docs/source/contributing/setup.md

```diff
@@ -173,3 +173,25 @@ python3 setup.py js    # fetch updated client-side js
 python3 setup.py css   # recompile CSS from LESS sources
 python3 setup.py jsx   # build React admin app
 ```
+
+### Failed to bind hub to http://127.0.0.1:8081/hub/
+
+This error can happen when there's already an application or a service using this port. Use the following command to find out which one it is.
```

Comment:
Worth pointing out that this applies to any port, not just 8081.

Reply:
Good point, I reworded and it made the hub an example.